app/src/main/res/values/styles.xml

```diff
@@ -29,8 +29,10 @@
         <item name="android:paddingEnd">16dp</item>
         <item name="android:textColor">@drawable/primary_button_text</item>
         <item name="android:textSize">16sp</item>
-        <item name="fontFamily">sans-serif-medium</item>
+        <item name="fontFamily">sans-serif</item>
```

Comment:
This is effectively changing all buttons using this style. Did you do so because of the mocks? I would add this value locally where needed.

Reply:
I did so, because all primary buttons (I could find) are using this `fontFamily` and `textStyle` bold.